src/mapper.rs

```diff
@@ -190,6 +190,26 @@ impl<'s> ProguardMapper<'s> {
         self.classes.get(class).map(|class| class.original)
     }
 
+    /// Remaps an obfuscated Class Method.
+    ///
+    /// The `class` argument has to be the fully-qualified obfuscated name of the
+    /// class, with its complete module prefix.
+    ///
+    /// The `method` will be resolved if that can be done unambiguously,
+    /// otherwise `None` is being returned.
+    pub fn remap_method(&'s self, class: &str, method: &str) -> Option<&'s str> {
+        let class = self.classes.get(class)?;
+        let mut members = class.members.get(method)?.iter();
+        let first = members.next()?;
+
+        // We conservatively check that all the mappings point to the same method,
+        // as we don’t have line numbers to disambiguate.
+        // We could potentially skip inlined functions here, but lets rather be conservative.
+        let all_matching = members.all(|member| member.original == first.original);
+
+        all_matching.then_some(first.original)
```

Comment:
This only remaps the method name but we'd need to remap the class name with it. Is the suggestion then to do the 2 remappings separating on the python side?

Reply:
👍🏻 very good idea, I changed it to return both.